src/bson/bson-config.h

```diff
@@ -1,26 +1,36 @@
-#ifndef BSON_CONFIG_H
-#define BSON_CONFIG_H
-
-#define PY_SSIZE_T_CLEAN  /* Make "s#" use Py_ssize_t rather than int. */
-
 /*
- * Rely on CPython to make libbson portable
+ * Copyright 2018-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
  */
-#include <Python.h>
```

Comment:
We need to keep this include and all the logic below that depends on it.

Reply:
Looks like all the `bson-config.h` changes were deleted. Was that intentional?

Reply:
That was the only way I was able to find to get it to work.

Reply:
We need to add the new libbson macros to bson-config.h. For now let's just define all the new macros to be 0. Then we can investigate how to set each one correctly based on the platform information present in `Python.h`. This might require some research on your end to figure out where each macro is used and which value would be the most portable. We can also remove `BSON_PTHREAD_ONCE_INIT_NEEDS_BRACES` which is no longer used in libbson.

Reply:
I believe I have updated the macros to be most portable to the best of what I could find online.